spec/driver_spec.rb

```diff
@@ -2674,8 +2676,11 @@ def which_for(character)
       end
 
       it "can allow specific hosts" do
-        driver.allow_url("example.com")
-        driver.allow_url("www.example.com")
+        configure do |config|
```

Comment:
Our specs have become sort of a DSL of their own on top of RSpec (`driver_for_*`, etc). I think they're generally in a pretty bad state (`driver_spec.rb` in particular). I can't think of a better way to mix configuration into the existing driver bootstrap methods, though.

I'd like to take a pass at reorganizing the specs in general. I think they do need to have a sort of specialty DSL because of the overhead in booting drivers, etc, but I'd like to at least make them more consistent.

However, I think all that has to take place outside this pull request.